src/main/java/org/folio/inventory/dataimport/handlers/actions/MarcBibModifiedPostProcessingEventHandler.java

```diff
@@ -82,6 +83,8 @@ public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload d
       Context localTenantContext = EventHandlingUtil.constructContext(dataImportEventPayload.getTenant(), dataImportEventPayload.getToken(), dataImportEventPayload.getOkapiUrl());
       Context targetInstanceContext = EventHandlingUtil.constructContext(getTenant(dataImportEventPayload), dataImportEventPayload.getToken(), dataImportEventPayload.getOkapiUrl());
       Promise<Instance> instanceUpdatePromise = Promise.promise();
+      String userId = dataImportEventPayload.getAdditionalProperties().get("userId").toString();
```

Comment:
@Maksat-Galymzhan  Please move "userId" to the constant.

Reply:
done